prettier.config.js

```diff
@@ -12,5 +12,5 @@ module.exports = {
   requirePragma: false,
   insertPragma: false,
   proseWrap: 'preserve',
-  htmlWhitespaceSensitivity: 'ignore',
+  htmlWhitespaceSensitivity: 'ignore'
```

Comment:
Revert this please (I guess you needed to run prettier).

Sorry, these comments are really lame that should be part of CI...